B827EBFFFEA6343D.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFEA6343D",
+    "servers": [
+      {
+        "server_address": "ttn-router-asia-se",
```

Comment:
This is not the correct router address. Check the list of existing routes here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses